osi_sensorviewconfiguration.proto

```diff
@@ -497,99 +452,73 @@ message LidarSensorViewConfiguration
     //
     optional MountingPosition mounting_position_rmse = 3;
 
-    // Field of View in horizontal orientation of the physical sensor.
+    // Emitter wavelength.
     //
-    // Viewing range: [- \c #field_of_view_horizontal/2,  \c
-    // #field_of_view_horizontal/2] azimuth in the sensor frame as defined in \c
-    // Spherical3d.
+    // This information can be used by a rendering engine to calculate
+    // signal interactions within the scene.
     //
-    // Unit: rad
-    optional double field_of_view_horizontal = 4;
-
-    // Field of View in vertical orientation of the physical sensor.
+    // Unit: nm
     //
-    // Viewing range: [- \c #field_of_view_vertical/2,  \c
-    // #field_of_view_vertical/2] elevation in the sensor frame at zero azimuth
-    // as defined in \c Spherical3d.
     //
-    // Unit: rad
-    optional double field_of_view_vertical = 5;
+    optional double emitter_wavelength = 4;
 
-    // Number of rays to cast across horizontal field of view.
+    // Emitted spatial signal gain per beam.
     //
-    // \note This is a characteristic of the ray tracing engine of the
-    // environment simulation, not a direct characteristic of the sensor.
+    // This describes the directions and signal gain of the emitted lidar beams,
+    // for which signal is to be expected from the simulated scene.
+    // Direction of beam center is given by the angles in the message
+    //
+    // \note Data is in sensor coordinate system
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
     //
-    optional uint32 number_of_rays_horizontal = 6;
+    repeated SpatialSignalGain emitted_beam = 5;
 
-    // Number of rays to cast across vertical field of view.
+    // Time offset for each emitted signal.
     //
-    // \note This is a characteristic of the ray tracing engine of the
-    // environment simulation, not a direct characteristic of the sensor.
+    // The time offset in microseconds of every emitted signal
+    // from each frame timestamp.
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
+    // Unit: s
     //
-    optional uint32 number_of_rays_vertical = 7;
+    // \note Usually, this is needed when ray tracing or similar techniques are applied.
+    //
+    // The time_offset_emission_size() must be the same as the emitted_signal_size().
+    //
+    repeated uint32 time_offset_emission = 6;
 
-    // Maximum number of interactions to take into account.
+    // Horizontal beam divergence
     //
-    // \note This is a characteristic of the ray tracing engine of the
-    // environment simulation, not a direct characteristic of the sensor.
+    // The beam divergence in horizontal direction.
     //
-    // \rules
-    // is_greater_than_or_equal_to: 1
-    // \endrules
+    // Unit: rad
     //
-    optional uint32 max_number_of_interactions = 8;
+    optional double beam_divergence_horizontal = 7;
```

Comment:
beam_divergence is typically a parameter in the datasheet of the lidar sensor. In the case of radar the continious character of the electromagnetic wave together with the antenna diagramm characteristics define the beam_divergence. Therefore this field should not be added to radar. In case of ultrasonic I'm not shure. Can you help @PhRosenberger?